scripts/bert/dataset.py

```diff
@@ -28,28 +34,424 @@ class MRPCDataset(TSVDataset):
     Parameters
     ----------
     segment : str or list of str, default 'train'
-        Dataset segment. Options are 'train', 'val', 'test' or their combinations.
+        Dataset segment. Options are 'train', 'dev', 'test' or their combinations.
     root : str, default '$GLUE_DIR/MRPC'
         Path to the folder which stores the MRPC dataset.
         The datset can be downloaded by the following script:
         https://gist.github.com/W4ngatang/60c2bdb54d156a41194446737ce03e2e
     """
+    task_name = 'MRPC'
+    is_pair = True
 
-    def __init__(self, segment='train',
-                 root=os.path.join(os.getenv('GLUE_DIR', 'glue_data'), 'MRPC')):
+    def __init__(self,
+                 segment='train',
+                 root=os.path.join(
+                     os.getenv('GLUE_DIR', 'glue_data'), task_name)):
         self._supported_segments = ['train', 'dev', 'test']
         assert segment in self._supported_segments, 'Unsupported segment: %s' % segment
         path = os.path.join(root, '%s.tsv' % segment)
         A_IDX, B_IDX, LABEL_IDX = 3, 4, 0
         fields = [A_IDX, B_IDX, LABEL_IDX]
-        super(MRPCDataset, self).__init__(path, num_discard_samples=1, field_indices=fields)
+        super(MRPCDataset, self).__init__(
+            path, num_discard_samples=1, field_indices=fields)
+
+    @staticmethod
+    def get_labels():
+        """Get classification label ids of the dataset."""
+        return ['0', '1']
+
+    @staticmethod
+    def get_metric():
+        """Get metrics Accuracy and F1"""
+        metric = CompositeEvalMetric()
+        for child_metric in [Accuracy(), F1()]:
+            metric.add(child_metric)
+        return metric
+
+
+class GLUEDataset(TSVDataset):
+    """GLUEDataset class"""
```

Comment:
When these classes are moved into gluonnlp data API, the API doc will need to be added. Doesn't have to be addressed in this PR.